app/javascript/containers/OptionsContainer.jsx

```diff
@@ -127,41 +129,40 @@ class OptionsContainer extends React.Component {
       }
     })();
 
-    return (
-      <div className="question__block">
-        <div
-          className={`question \
-          ${this.state.selected ? "question--selected" : ""} \
-          ${
-            this.props.answer && this.props.answer.error
-              ? "question--error"
-              : ""
-          }`}
-        >
-          <p>{this.props.text}</p>
-          {optionsComponent}
-          <a onClick={this.onClickHandoff.bind(this)}>Handoff</a>
-          <Status
-            fetchObject={this.props.answer}
-            onRetry={this.onRetry.bind(this)}
-          />
-        </div>
-        {this.props.answer ? (
-          <DependentQuestions
-            answer={this.props.answer}
-            dependentQuestions={this.props.dependentQuestions}
-            buildingId={this.props.building_id}
-            parentIsHidden={this.props.parentIsHidden}
-            disableFocusOnMount={this.props.question_type == "RangeOption"}
-          />
-        ) : null}
+    return (<div className="question__block">
+      <div
+        className={
```

Comment:
The previous comment I made using the `classNames()` syntax was when you were using `getStyle()` to fetch styles. I think this syntax here is nice as well and it's up to you which you prefer, but for consistency might be good to change this to `classNames` syntax or change the rest to this syntax